src/content/release/breaking-changes/deprecate-inputdecoration-maintainhintheight.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Deprecate `InputDecoration.maintainHintHeight` in favor of
+  `InputDecoration.maintainHintSize`
+description: >-
+  The `InputDecoration.maintainHintHeight` parameter has been replaced by
+  `InputDecoration.maintainHintSize`.
+---
+
+## Summary
+
+The [`InputDecoration.maintainHintHeight`][] parameter was deprecated
+in favor of the [`InputDecoration.maintainHintSize`][] parameter.
```

Comment:
Right now, this is going to a non-existent page. It looks like the following page needs to be added first: https://api.flutter.dev/flutter/material/InputDecoration/maintainHintSize.html

If you need help, let me know.

Reply:
Maybe link to https://main-api.flutter.dev/flutter/material/InputDecoration/maintainHintSize.html?

Reply:
Thanks for the review 🙏 
I changed the link to https://main-api.flutter.dev/flutter/material/InputDecoration/maintainHintSize.html.
Is it ok?